react-js/react.md

```diff
@@ -4,21 +4,22 @@
 8 Hours
```

Comment:
Please break this up by section and have an estimation for each.  See our template for guidance: https://github.com/Techtonica/curriculum/blob/master/_templates/topic-outline.md

Reply:
Done